marlowe/doc/playground-blockly.adoc

```diff
@@ -3,10 +3,79 @@
 
 == Marlowe in Blockly
 
-So far in these tutorials we have concentrated on building contracts in the textual version of Marlowe, embedded in Haskell, Marlowe contracts can also be built using the Blockly visual programming environment, as described in this series of videos.
+So far in these tutorials we have concentrated on building contracts in the textual version of Marlowe, embedded in Haskell, Marlowe contracts can also be built using the Blockly visual programming environment, as we describe here.
 
+=== Getting started
+
+To start building a Blockly project in the Marlowe Playground, which you can find out more about in the section about xref:playground-overview#playground-overview[], select *New Project* and then *Blockly* for the project type. You'll now see a window like this:
+
+image:blockly1.png[Blockly window]
+
+Contracts are assembled by adding components to the holes in the blocks. To build a contract, we have to fill the single, top-level, hole here with a `Contract`, and blocks for these are found by clicking on *Contracts* in the menu on the left hand side. Doing this shows all the different blocks that build contracts, as shown here:
+
+image:blockly2.png[Contracts in Blockly]
+
+We can select one and drag it into the pane, like this:
+
+image:blockly3.png[A contract block]
+
+and then fit it into the top-level hole, so that we're constructing a contract with a `When` as its main construct.
+
+image:blockly4.png[Fitting in a contract block]
+
+This is the process that we repeat, adding contracts and other types of components – all shown in the menu on the left-hand side – to build a complete contract.
+
+Let's fill in the timeout information and put a `Close` contract to perform on timeout:
```

Comment:
"add" a `Close` contract "to be performed in case of" timeout?

Reply:
done